citrixitm/resource_citrixitm_dns_app_test.go

```diff
@@ -161,7 +173,9 @@ func testAccCheckCitrixITMDnsAppAttributes(got *itm.DNSApp, want *testAccCitrixI
 		if err = testValues("fallback CNAME", want.FallbackCname, got.FallbackCname); err != nil {
 			return
 		}
-		if err = testValues("app data", want.AppData, got.AppData); err != nil {
+		// App data is stripped of leading and trailing whitespace before being submitted to the API
+		trimmed := strings.TrimSpace(want.AppData)
+		if err = testValues("app data", trimmed, got.AppData); err != nil {
```

Comment:
I don't know what the convention is in js but since `trimmed` is reference only once, it could perhaps be `testValues("app data", strings.TrimSpace(want.AppData), .. `?

Reply:
Oh, good point. I had some logging earlier that used it too, but I removed it. The suggested change makes sense.